_data/projects/Learning_Object_Oriented_Python.yml

```
@@ -0,0 +1,10 @@
+name: Learning-Object_Oriented-Python
+desc: This repository walks you through the Object Oriented Programming in Python.
+site: https://www.github.com/josharsh/Learning-Object-Oriented-Python
+tags:
+- python
+- documentation
+- Object-Oriented-Python
```

The CI build is currently failing because we want all tags to be lower-case characters:

```suggestion
- object-oriented-python
```